templates/file_crash_bug_description.md.jinja

```diff
@@ -28,7 +28,7 @@ Top {{ frames|length }} frames of crashing thread:
 {% endif %}
 
 
-Here are some insights about the crash signature based on recent data at the time of reporting the bug:
+Here are some insights about the crash signature based on the last {{ signature_details_delta }} of {{ signature_details_channel }} data:
```

Comment:
When the signature is new, e.g. just started 2 days ago, it'd be confusing to say "based on the last 10 months of ...". Could we check when the signature started and replace the "10 months" with that if it's later than 10 months ago?

Reply:
I agree it can be confusing. I do not feel changing the number will make it clearer. Maybe rewording can make things less confusing. What do you think about the following rewarding?


> By querying Nigtly crashes reported within the last 2 months, here are some insights about the signature:

We could even make it more explicit by adding a list item about the start date: 

> - First crash report: 2023-07-10

Reply:
Rewording and making it more explicit sounds good to me!